packages/app/src/App.tsx

```diff
@@ -289,7 +273,7 @@ function App(props: { flameFromQuery?: FlameFunction[] }) {
               formatValue={(value) => value.toString()}
             />
             <label class={ui.labeledInput}>
-              Enable adaptive filter
+              Adaptive filter
```

Comment:
The users would  know intuitively what does this do? Is there a more user friendly name for it?

Reply:
Idk, just wanted to remove `enable` for now